src/lib/app-state-hoc.jsx

```diff
@@ -71,7 +71,7 @@ const AppStateHOC = function (WrappedComponent) {
             return (
                 <Provider store={this.store}>
                     <IntlProvider>
-                        <ErrorBoundary>
+                        <ErrorBoundary action="Top Level App">
```

Comment:
I know Action can be anything, but I'm wondering if there are conventions that would make it more useful for filtering in analytics

Reply:
At this point only that they be different from each other and meaningful.